client/state/sites/selectors/get-jetpack-admin-url.js

```diff
@@ -0,0 +1,15 @@
+import { format as formatUrl, getUrlParts, getUrlFromParts } from '@automattic/calypso-url';
+import getSiteAdminPage from './get-site-admin-page';
+import getSiteAdminUrl from './get-site-admin-url';
+
+export default function getJetpackAdminUrl( state, siteId ) {
+	const siteAdminUrl = getSiteAdminUrl( state, siteId );
+	if ( null === siteAdminUrl ) {
```

Comment:
I know this wasn't your doing, but in the name of leaving it better than we found it, let's re-write this conditional so it's not Yoda-style:

```suggestion
	if ( siteAdminUrl === null ) {
```

Reply:
I see you and I raise you` if ( ! siteAdminUrl )`

Reply:
@TheSteveK it's better to be precise, so I'd stick with `if ( siteAdminUrl === null )` because function returns `string | null`. Not to this example, but there are situations where `if ( ! siteAdminUrl )` would give you the wrong result (in case of numbers and 0 for example).

Reply:
Ok, I changed it back, but it's worth mentioning most of the other selectors use ! (typically !siteId)